tianshou/data/buffer.py

```diff
@@ -359,6 +379,22 @@ def __getitem__(
             policy=self.get(index, "policy"),
         )
 
+    def save_hdf5(self, path: str) -> None:
+        """Save replay buffer to HDF5 file."""
+        with h5py.File(path, "w") as f:
+            to_hdf5(self.__getstate__(), f)
+
+    @classmethod
+    def load_hdf5(
+        cls, path: str, device: Optional[str] = None
+    ) -> "ReplayBuffer":
+        """Load replay buffer from HDF5 file."""
+        with h5py.File(path, "r") as f:
+            buf = cls.__new__(cls)
+            buf_state = from_hdf5(f, device=device)
+            buf.__setstate__(buf_state)
```

Comment:
Why not `buf.__setstate__(from_hdf5(f, device=device))` as you did for `save_hdf5` but reversed ?

Reply:
Sorry, I don't understand. Isn't this what I have done?

Reply:
Here you wrote it in 2 lines of code, whereas you wrote it in 1 in `save_hdf5`. Just a small detail to be consistent between both methods.

Reply:
Ah, I see. I will put everything into one line, thanks.